app/models/user.rb

```diff
@@ -5,5 +5,4 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   # Validates
   validates :email, presence: true
-  validates :permission, presence: true
```

Comment:
tại sao lại bỏ validation này?

Reply:
validates này sau mình vẫn dùng mà anh.

Reply:
@tuanhv-0694 @thiennlinh 
lúc đăng kí đang báo lỗi phần permission 
nên a sẽ tạo PR mới để xử lý phần validation

Reply:
ở lỗi gì anh? Đăng kí thì phải xem nó có quyền gì luôn chứ ạ??

Reply:
@tuanhv-0694 CC: @thiennlinh 

HIện tại khi user đăng kí account, a đang để không có trường nhập permission.
Nên ở PR này, a tạm thời xóa validate của permission.
để khi lưu thì sẽ lưu mặc định là nil.
--
A dự định là Ở phía admin.
Admin sẽ vào phần quản lý user và add thêm user và set permission trong đó.
và sẽ thực hiện validate permission sau.

Reply:
@dungdd-1735 nếu thế thì CSDL mình lại phải có một bảng cho admin à a, vì nó là 1 usecase

Reply:
em đang hiểu là permission ở đây rất giống với user_type của bên group 1, vậy sao anh không làm gióng như bên đó ạ

Reply:
@tuanhv-0694 
à cũng tương tự user_type.
Nhưng nếu để validates :permission, presence: true thì lúc đăng kí báo lỗi Permission can't be blank
--
giờ mình set giá trị mặc định cho nó luôn nhỉ ?

Reply:
@dungdd-1735 luc dau em co de mot input value la permision day a

Reply:
@dungdd-1735 @thiennlinh làm như linh bảo ý anh, cho họ chọn luôn là user type nào, nhà bán hàng hay là người mua hàng ạ :D

Reply:
@tuanhv-0694  @thiennlinh 
à trong commit mới a set default user type là customer rồi nha